### ISSUE TITLE
View elements loose elevation on transition start on Android Material. Leads to visual flicker.

### ISSUE THREAD

**Issue by sserdyuk** [NativeScript/NativeScript#4416]
I have navigating between two views that have the same action bar content and look. I'm using fade animation by default, and I noticed unpleasant flicker on navigation. What I found upon further research is that the elevation of elements on the view that I am navigating from is removed upon beginning of the transition. The issue is specific to Android with API level 21+.

Here's the example of action bar flicker on fade navigation and a slow-mo version with a slide transition to make it more obvious.

```
import { Frame } from "ui/frame";
// Frame.defaultTransition = { name: "slide", duration: 5000 };
Frame.defaultTransition = { name: "fade" };

```

```
import { topmost } from "ui/frame";

export function onTap(args: EventData) {
    topmost().navigate("main-page")
}
```

```
<Page xmlns="http://schemas.nativescript.org/tns.xsd" navigatingTo="navigatingTo" class="page">
    <Page.actionBar>
        <ActionBar title="My App" icon="" class="action-bar">
        </ActionBar>
    </Page.actionBar>
    <StackLayout class="p-20">
        <Label text="Tap the button" class="h1 text-center"/>
        <Button text="TAP" tap="onTap" class="btn btn-primary btn-active"/>
    </StackLayout>
</Page>

```

```
$ tns info
All NativeScript components versions information
┌──────────────────┬─────────────────┬────────────────┬───────────────┐
│ Component        │ Current version │ Latest version │ Information   │
│ nativescript     │ 3.0.3           │ 3.0.3          │ Up to date    │
│ tns-core-modules │ 3.0.1           │ 3.0.1          │ Up to date    │
│ tns-android      │ 3.0.1           │ 3.0.1          │ Up to date    │
│ tns-ios          │                 │ 3.0.1          │ Not installed │
└──────────────────┴─────────────────┴────────────────┴───────────────┘
```

![giphy 2](https://user-images.githubusercontent.com/395998/27261954-b44ef658-541b-11e7-92f1-f840eba8bc15.gif)

![giphy 1](https://user-images.githubusercontent.com/395998/27261956-b7fa05e0-541b-11e7-9d41-11cc066e968d.gif)

**Comment by NickIliev** [comment]
@sserdyuk thank you for the detailed description of this issue! I can confirm taht indeed the page transitions are removing the elevation shadow during the transition,

Steps to reproduce:
- use this [test applicaiton](https://github.com/NickIliev/NativeScript-Issues-2017/tree/master/NativeScript/issue_4416).
- navigate from main-page to sub-page and back.
- on each navigation, a page transition is applied.

Result: the ActionBar is loosing its elevation shadow on transition started.

**Comment by sserdyuk** [comment]
For the record, it's not just the ActionBar. The button looses elevation too.

**Comment by tsonevn** [comment]
HI @sserdyuk,
The issue seems to be fixed with the latest tns-core-modules `4.1.1`. 
[Archive.zip](https://github.com/NativeScript/NativeScript/files/2235503/Archive.zip)
